test/sqs_test.py

```diff
@@ -59,3 +59,7 @@ def test_delete_messages(self):
         # let's leave it at this minimal state.
         messages = self._sqs.get_messages(self.TEST_QUEUE_NAME)
         self._sqs.delete_messages(self.TEST_QUEUE_NAME, messages)
+
+    def test_send_message(self):
```

Comment:
Docstring, please.

Also, if you are not implementing a full unit test for `send_message()`, please leave a TODO to fix later.